src/cascadia/WindowsTerminal/AppHost.cpp

```diff
@@ -1233,9 +1233,10 @@ void AppHost::_SummonWindowRequested(const winrt::Windows::Foundation::IInspecta
 }
 
 void AppHost::_OpenSystemMenu(const winrt::Windows::Foundation::IInspectable&,
-                              const winrt::Windows::Foundation::IInspectable&)
+                              const winrt::Windows::Foundation::IInspectable&) const
 {
-    _window->OpenSystemMenu(std::nullopt, std::nullopt);
+    const NonClientIslandWindow* nonClientWindow{ static_cast<NonClientIslandWindow*>(_window.get()) };

```

Comment:
What happens if the user doesn't have "Show tabs in titlebar" enabled? Won't this explode, since `this` won't be a `NonClientIslandWindow`?

Reply:
Didn't know about this case. Let me check and iterate. Thank you for your feedback.